src/share/util/shr_orb_mod.F90

```diff
@@ -433,7 +433,7 @@ SUBROUTINE shr_orb_params( iyear_AD , eccen  , obliq , mvelp     ,     &
     real   (SHR_KIND_R8) :: obsum   ! Obliquity series summation
     real   (SHR_KIND_R8) :: cossum  ! Cos series summation for eccentricity/fvelp
     real   (SHR_KIND_R8) :: sinsum  ! Sin series summation for eccentricity/fvelp
-    real   (SHR_KIND_R8) :: fvelp   ! Fixed vernal equinox long of perihelion
+    real   (SHR_KIND_R8) :: fvelp = 0.0_SHR_KIND_R8   ! Fixed vernal equinox long of perihelion
```

Comment:
This initialization should be done in the code, not in the variable declaration: doing this in the variable declaration implicitly gives it the `save` attribute, which is generally a bad thing.

So please move this initialization to just before this block of code:

```fortran
       if (abs(cossum) .le. 1.0E-8_SHR_KIND_R8) then
          if (sinsum .eq. 0.0_SHR_KIND_R8) then
             fvelp = 0.0_SHR_KIND_R8
          else if (sinsum .lt. 0.0_SHR_KIND_R8) then
             fvelp = 1.5_SHR_KIND_R8*pi
          else if (sinsum .gt. 0.0_SHR_KIND_R8) then
             fvelp = .5_SHR_KIND_R8*pi
          endif
       else if (cossum .lt. 0.0_SHR_KIND_R8) then
          fvelp = atan(sinsum/cossum) + pi
       else if (cossum .gt. 0.0_SHR_KIND_R8) then
          if (sinsum .lt. 0.0_SHR_KIND_R8) then
             fvelp = atan(sinsum/cossum) + 2.0_SHR_KIND_R8*pi
          else
             fvelp = atan(sinsum/cossum)
          endif
       endif
```

or (better in my mind) change this line:

```fortran
       else if (cossum .gt. 0.0_SHR_KIND_R8) then
```

to:

```fortran
       else  ! cossum > 1.e-8
```

(I'm pretty sure that the logic ends up being the same with that change, though you could double-check it. It makes it more clear to the reader - as well as the compiler- that one of those conditional blocks will always be executed.)